tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -816,22 +815,10 @@ export class WebGPUBackend extends KernelBackend {
     return timeElapsedNanos / 1000000;
   }
 
-  private getCPUBackend(): KernelBackend|null {
-    if (!env().getBool('WEBGPU_CPU_FORWARD')) {
-      return null;
-    }
-
-    if (this.cpuBackend == null) {
-      this.cpuBackend = engine().findBackend('cpu');
```

Comment:
We shouldn't get cpu backend from `engine().findBackend('cpu')`. Otherwise, cpu backend is never used if cpu is not registered explicitly. Currently, cpu impl is imported to each backend. Refer to #4908